input/cql/PCSBPScreeningFollowUpFHIR.cql

```diff
@@ -278,9 +288,12 @@ define "Encounter with Order for Hypertension Follow Up Declined by Patient":
       union ( "Encounter with First Hypertensive Reading SBP Greater than or Equal to 130 OR DBP Greater than or Equal to 80" FirstHTNEncounter
           with ( [ServiceNotRequested: "Follow Up Within 4 Weeks"]
             union [ServiceNotRequested: "Referral to Primary Care or Alternate Provider"] ) FirstHTNDeclinedInterventions
-            such that FirstHTNDeclinedInterventions.reasonRefused in "Patient Declined"
-              and FirstHTNDeclinedInterventions.authoredOn during day of FirstHTNEncounter.period
+            such that FirstHTNDeclinedInterventions.authoredOn during day of FirstHTNEncounter.period
               and FirstHTNDeclinedInterventions.status = 'completed'
+              and exists(FirstHTNDeclinedInterventions.extension E 
+                where E.url = 'http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-doNotPerformReason' 
+                and E.value as Code in "Patient Declined"
+              )
```

Comment:
I think this is expected to work as `FirstHTNDeclinedInterventions.reasonRefused`